### PR TITLE
[Docathon][Add CN Doc No.64-67]

### PR DESCRIPTION
**Pull request by ooooo-create**
add
- `paddle.incubate.asp.reset_excluded_layers`
- `paddle.incubate.asp.set_excluded_layers`
- `paddle.incubate.asp.add_supported_layer`
- `paddle.incubate.asp.add_supported_layer`

link #6193 

en_doc
- [https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/incubate/asp/reset_excluded_layers_en.html](https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/incubate/asp/reset_excluded_layers_en.html)
- [https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/incubate/asp/set_excluded_layers_en.html](https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/incubate/asp/set_excluded_layers_en.html)
- [https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/incubate/asp/add_supported_layer_en.html](https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/incubate/asp/add_supported_layer_en.html)
- [https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/incubate/asp/calculate_density_en.html](https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/incubate/asp/calculate_density_en.html)

@sunzhongkai588 @mingxu1067

### CODE REVIEW

File: docs/api/paddle/incubate/asp/add_supported_layer_cn.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+-------------------------------
+
+.. py:function:: paddle.incubate.asp.add_supported_layer(layer, pruning_func=None)
+
+添加支持的 layer 及其相应的剪枝函数。
+
+参数
+:::::::::
+    - **name** (string|Layer) - 需要支持的 layer 的名称或类型。 如果 layer 是 Layer，那么它将在内部转为字符串。ASP 将使用这个名称来匹配参数的名称并调用其相应的剪枝函数。
+    - **pruning_func** (function，可选) - 一个接收五个参数（weight_nparray、m、n、func_name、param_name）的函数类型，weight_nparray 是 nparray 格式的权重，param_name 是权重的名称，m、n 和 func_name，详细信息请参见 :ref:`cn_api_paddle_incubate_asp_prune_model`。
```
Comment (sunzhongkai588) [inline docs/api/paddle/incubate/asp/add_supported_layer_cn.rst:13]:
```suggestion
    - **pruning_func** (function，可选) - 一个接收五个参数（weight_nparray、m、n、func_name、param_name）的函数类型，weight_nparray 是 nparray 格式的权重，param_name 是 weight、m、n 、func_name 的名称，详细信息请参见 :ref:`cn_api_paddle_incubate_asp_prune_model`。
```

File: docs/api/paddle/incubate/asp/reset_excluded_layers_cn.rst [hunk 2]
```diff
@@ -0,0 +1,26 @@
+.. _cn_api_paddle_incubate_asp_reset_excluded_layers:
+
+reset_excluded_layers
+-------------------------------
+
+.. py:function:: paddle.incubate.asp.reset_excluded_layers(main_program=None)
+
+
+重置与 main_program 对应的 excluded_layers 设置。如果 main_program 为 None，则 excepted_layers 的所有配置都将被清除。
```
Comment (sunzhongkai588) [inline docs/api/paddle/incubate/asp/reset_excluded_layers_cn.rst]:
```suggestion
重置与 `main_program` 对应的 excluded_layers 设置。如果 `main_program` 为 None，则 excepted_layers 的所有配置都将被清除。
```

File: docs/api/paddle/incubate/asp/reset_excluded_layers_cn.rst [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+
+参数
+::::::::::::
+
+- **main_program** (Program，可选) - 包含模型定义及其参数的 Program。如果给出 None，那么这个函数将重置所有 exclusion_layers。 默认为 None。
```
Comment (sunzhongkai588) [inline docs/api/paddle/incubate/asp/reset_excluded_layers_cn.rst]:
```suggestion
- **main_program** (Program，可选) - 包含模型定义及其参数的 Program。如果为 None，那么这个函数将重置所有 exclusion_layers。 默认为 None。
```

File: docs/api/paddle/incubate/asp/set_excluded_layers_cn.rst [hunk 4]
```diff
@@ -0,0 +1,26 @@
+.. _cn_api_paddle_incubate_asp_set_excluded_layers:
+
+set_excluded_layers
+-------------------------------
+
+.. py:function:: paddle.incubate.asp.set_excluded_layers(param_names, main_program=None)
+
+
+设置不会被修剪为稀疏权重的 layer 的参数名称。
```
Comment (sunzhongkai588) [inline docs/api/paddle/incubate/asp/set_excluded_layers_cn.rst]:
```suggestion
设置不会被裁剪为稀疏权重的 layer 的参数名称。
```

File: docs/api/paddle/incubate/asp/add_supported_layer_cn.rst (hunk 1 above)
Comment (ooooo-create) [reply docs/api/paddle/incubate/asp/add_supported_layer_cn.rst:13]: 图片
这个看着似乎 `param_name` 仅仅是 `weight` 的名称

**Comment by sunzhongkai588** [comment]
@SigureMo 一师傅看看..为啥 copy-from 都不成功。

**Review by sunzhongkai588 — APPROVED** [review]
LGTM

**Pull request merged**